src/media/AvatarMember.jsx

```diff
@@ -30,16 +31,22 @@ class AvatarMember extends React.PureComponent {
 			className
 		);
 
-		const noPhotoImage = require('base64-image-loader!swarm-icons/dist/optimized/profile.svg');
+		const noPhotoIcon = (<Icon
+			shape="profile"
+			size="m"
+		/>);
 
-		return (
-			<Avatar
-				alt={member.name}
-				src={showNoPhoto ? noPhotoImage : member.photo[photoLink]}
-				className={classNames}
-				{...other}
-			/>
-		);
+		const allProps = {
+			alt: member.name,
+			src: !showNoPhoto && member.photo[photoLink],
```

Comment:
I believe you'll get a proptype error when you pass `src=false` in the `showNoPhoto` case
https://github.com/meetup/meetup-web-components/blob/master/src/media/Avatar.jsx#L84